core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2019, 2021 Contributors to the Eclipse Foundation
+ * Copyright (c) 2019, 2021, 2023 Contributors to the Eclipse Foundation
```

Comment:
we indicate the range by means of specifying the year of initial creation and the year of the latest change. So in this case `2019, 2023`. Can you please also change this in the other classes accordingly?

Reply:
Done